src/Illuminate/Foundation/Console/ModelMakeCommand.php

```diff
@@ -61,6 +61,7 @@ public function handle()
             $this->input->setOption('seed', true);
             $this->input->setOption('migration', true);
             $this->input->setOption('controller', true);
+            $this->input->setOption('observer', true);
```

Comment:
I don't think this should be part of the "all" flag. I doubt many people are relying on observers.

Reply:
In my opinion, I have added this line with the `--all` option because the **Observer** must be related to a model so when I pass the `--all` everything related to the model must be created.